malariagen_data/anoph/cnv_params.py

```diff
@@ -20,3 +20,6 @@
     `coverage_calls_analysis_ids` property for available values.
     """,
 ]
+
+# This is grey followed by PuOr5
+colorscale_default = ["#cccccc", "#5e3c99", "#b2abd2", "#f7f7f7", "#fdb863", "#e66101"]
```

Comment:
Nice to have this defined in a common location. Could we also now use this within the plot_cnv_hmm_heatmap() function so we know the same colours for CNV states are being used throughout?